Dockerfile

```diff
@@ -14,8 +14,9 @@ RUN apk add --update bash curl git openssl python3 terraform
 RUN pip3 install pre-commit
 
 # Download Tflint
-ADD https://github.com/terraform-linters/tflint/releases/download/${TFLINT_VERSION}/tflint_linux_amd64.zip .
-ADD https://github.com/terraform-linters/tflint/releases/download/${TFLINT_VERSION}/checksums.txt .
+RUN wget \
```

Comment:
Why would you replace ADD through WGET here?

Reply:
To not redownload every run and tiger all later layers to be rebuild.. so it is 100% cachable

Reply:
agree on the change or revert?